.gitignore

```diff
@@ -21,3 +21,9 @@
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+# Ignore Gradle project-specific cache directory
+.gradle
+
+# Ignore Gradle build output directory
+build
```

Comment:
Need to add directories such as target/ to the gitignore. Refer https://github.com/ballerina-platform/module-ballerina-socket/blob/81d949c548c490b61ac646c11bf6e1029ca954f9/.gitignore#L27. 

Basically, when you run the Gradle build there should not be any automatically generated unversioned files. Those unversioned files failed the release process when we release through the release plugin.